src/lib/components/DropDownClasses.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+    import { createEventDispatcher } from "svelte";
+
+    const dispatch = createEventDispatcher();
+
+    let classes = ["all","artificer","barbarian","bard","cleric","druid","fighter","monk","paladin","ranger","rogue","sorcerer","warlock","wizard"];
+    let selectedClass = "all";
+
+    function emitUpdate(): void {
+        dispatch("classUpdate", { class:selectedClass });
+    }
+</script>
```

Comment:
Der findes allerde en enum med værdierne for classes i `$lib/constants/classes`. Har ikke tjekket om der mangler nogle, men du tilføjer bare hvis der gør. Med dén kan du bruge `Object.values(Class)` til at gøre koden lidt kortere - se forslag.

```suggestion
<script lang="ts">
    import { createEventDispatcher } from "svelte";
    import { Class } from "$lib/constants/classes";

    const dispatch = createEventDispatcher();

    let selectedClass = "All";
    let classes = [ "All", ...Object.values(Class) ];

    function emitUpdate(): void {
        dispatch("classUpdate", { class: selectedClass });
    }
</script>
```